### DISCUSSION TITLE
Does res_line["loading_percent"] refer to a 3-phase line?

### DISCUSSION THREAD

**Discussion by timebis** [e2nIEE/pandapower#1693] [Q&A]
This discution is related to #[this unactepted PR](https://github.com/e2nIEE/pandapower/pull/1645)

### Problem :

here is an extract of the Pandapower Basic Standard Types line :

<html>
<body>

 name | r_ohm_per_km | x_ohm_per_km | c_nf_per_km | max_i_ka | type | q_mm2 | alpha
-- | -- | -- | -- | -- | -- | -- | --
NAYY 4x50 SE | 0.642 | 0.083 | 210.0 | 0.142 | cs | 50 | 0.00403 <br class="Apple-interchange-newline">
</body>
</html>

here, `max_i_ka = 142A` correspond to the admissible current in one wire. This assumption is made according to classical constructor datasheet of thermal limit given per wire : 
![image](https://user-images.githubusercontent.com/48658392/187084954-8eb87d23-1450-4ba6-8455-2618b6d3fd81.png)

So, `max_i_ka` in pandapower is **not** the admissible current in one 3ph line system.

In the code we have : 
`    res_line_df["loading_percent"].values[:] = i_ka / i_max * 100
`
Where `i_ka` is considered as the equivalent current going through 3 wire.
![image](https://user-images.githubusercontent.com/48658392/187085657-ce0a826b-cfda-4629-8599-822e8ba4647c.png)
The fact that `i_ka` and `i_max_ka` are not for the same system (1 wire or the full cable) make the equation false.

Do you guys agree with me ?

### Suggestion :

I can do a PR to change :
`res_line_df["loading_percent"].values[:] = i_ka / i_max / 3 * 100
`

Best regard !

**Comment by jkisse** [comment]
A NAYY 4x50 cable already consists of 3 phases (+ 1 return), thus the **4**x50. [Here ](https://images.app.goo.gl/2xMGBh9oKtWk6h2e7) is a picture for clarification. It is just one cable - so, the values are correct, and no change is necessary.

**Reply by timebis** [reply]
Thanks for the remark but it's the point i'm trying to make.

For me, the limit (in A) given by constructors are given for 1*50.

**Comment by timebis** [comment]
I just understood my mistake..
I thought that `I_ka` was the equivalent current given for a single phase system. I don't why i got this in my mind, I made the calculation to verify it but I failed it ^^
Sorry for bothering.
